package.json

```diff
@@ -55,6 +55,9 @@
     "url-loader": "^1.0.1",
     "webpack": "^4.27.1",
     "webpack-cli": "^3.1.0",
-    "webpack-filter-warnings-plugin": "^1.2.1"
+    "webpack-filter-warnings-plugin": "^1.2.1",
+    "@angular-devkit/core": "~7.3.7",
```

Comment:
These seem like they are not in alpha order. Did you add them manually?

Reply:
Yeah, I added it manually and total forgot to order them, I'll update that.